[Docs] Purpose of using two dbs

slancerk opened the pull request:
## Why ToolJet Requires Two Databases

| **Environment Variable** | **Description** |
|---------------------------|-----------------|
| **TOOLJET_DB** | Stores ToolJet's internal metadata, including tables created within the platform |
| **PG_DB** | The primary database for storing application data, used by the apps built on ToolJet to manage end-user data. |

adishM98 commented on docs/versioned_docs/version-3.5.0-LTS/setup/env-vars.md:
```diff
@@ -48,6 +48,13 @@ Replace `username`, `password`, `hostname`, `port`, and `database_name` with you
 - `TOOLJET_DB_PASS`: Database password
 - `TOOLJET_DB_PORT`: Database port
 
+#### Why ToolJet Requires Two Databases
```
Could we remove this table format. It seem odd since others do not have table. It would be better if we can explain it without puting in table format.

slancerk commented:
Updated Azure container docs v3.5 
- Added custom Vnet integration in container apps.
<img width="315" alt="Screenshot 2025-05-18 at 10 21 47 PM" src="https://github.com/user-attachments/assets/2acc896f-d3a2-46b3-82ac-8e1d84ef90ca" />

RudraPratik30 commented on docs/versioned_docs/version-3.5.0-LTS/setup/azure-container.md:
```diff
@@ -31,7 +31,26 @@ ToolJet comes with a **built-in Redis setup**, which is used for multiplayer edi
  </div>
 
-3. In the container tab, uncheck the "Use quickstart image" option to select the image source manually.
+3. Select "Create new environment" to configure the basic networking setup.
```
The "Create new environment" button is not visible in the last screenshot, is it the "Create new" link below the Containers App Environment, or "Review + Create" button at the bottom or something else? can we add that?

slancerk replied:
Updated the image, now "Create new environment" is well visible.

RudraPratik30 commented on docs/versioned_docs/version-3.5.0-LTS/setup/azure-container.md:
```diff
@@ -31,7 +31,26 @@ ToolJet comes with a **built-in Redis setup**, which is used for multiplayer edi
 
-3. In the container tab, uncheck the "Use quickstart image" option to select the image source manually.
+3. Select "Create new environment" to configure the basic networking setup.
+ <div style={{textAlign: 'center'}}>
```
Not super important, but from the next time you don't need to use this div tag, you can directly use the img tag.

RudraPratik30 commented on docs/versioned_docs/version-3.5.0-LTS/setup/azure-container.md:
```diff
@@ -31,7 +31,26 @@ ToolJet comes with a **built-in Redis setup**, which is used for multiplayer edi
+
+ </div>
+   
+Now, let’s proceed to the Networking section for detailed configuration steps. Please update the configurations as shown below:
```
Can we make this as the 4th step? Also if we don't need to edit the workload profiles and monitoring page then can we mention that keep this to settings as default

RudraPratik30 commented on docs/versioned_docs/version-3.5.0-LTS/setup/azure-container.md:
```diff
@@ -31,7 +31,26 @@ ToolJet comes with a **built-in Redis setup**, which is used for multiplayer edi
+ <img className="screenshot-full" src="/img/setup/azure-container/step3-2.png" alt="Deploying ToolJet on Azure container apps" />
+
+ </div>
+Create and exit.
```
This is not visible below the screenshot, probably we can mark this as a different step, or rephrase the previous step and mention this there itself

slancerk replied:
It is a part of 4th step.
<img width="752" alt="Screenshot 2025-05-21 at 8 39 05 PM" src="https://github.com/user-attachments/assets/2e6e1e54-0081-4ca3-be34-b623853cc287" />

RudraPratik30 reviewed (approved):
looks good
